site/book/04-using-functions/01-declarative-function-execution.md

```diff
@@ -37,7 +37,7 @@ This declares two functions:
 - `kubeval` is an example validator function which you've seen before. It validates the resources
   against their OpenAPI schema.
 
-?> Refer to the [Functions Catalog][func-cat] for details on how to use a particular function.
+?> Refer to the [Functions Catalog](https://catalog.kpt.dev ':crossorigin :target=_self') for details on how to use a particular function.
```

Comment:
Doing this for each reference will be brittle as it's easy to forget crossorigin. Can you find a permanent solution? Also does it affect any external link or just the catalog?

Reply:
Yeah, we can mark the catalog (and other common external links) as safe cross origin links at the docsify level. I'll do that here.